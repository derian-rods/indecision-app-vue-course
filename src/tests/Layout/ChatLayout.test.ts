import { describe, it, expect } from 'vitest';
import { mount } from '@vue/test-utils';
import ChatLayout from '@/Layout/ChatLayout.vue';

const wrapper = mount(ChatLayout);

describe('ChatLayout Component', () => {
  it('should render the ChatLayout component', () => {
    expect(wrapper).toBeTruthy();
  });

  it('should match the snapshot', () => {
    expect(wrapper.html()).toMatchSnapshot();
  });
});
