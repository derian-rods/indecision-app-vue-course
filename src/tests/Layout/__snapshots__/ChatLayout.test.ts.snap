// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`ChatLayout Component > should match the snapshot 1`] = `
"<div class="bg-gray-100 h-screen flex flex-col max-w-lg mx-auto">
  <div class="bg-blue-500 p-4 text-white flex justify-between items-center"><span>Mom</span></div>
</div>"
`;
